source/connecting/oidc-to-oidc.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-conn-oidc2oidc:
```

Comment:
@pmeredit Wouldn't OIDC support work if one cluster is OIDC and the other isn't?

Reply:
Sure, but I do also think that would be a very rare use case for our users. Probably worth calling that out, however. Mixing and matching auth on the source and dest clusters is always viable.